.env

```diff
@@ -1,3 +1,3 @@
-HOST=192.168.3.2
-HTTPS=true
+HOST=localhost
```

Comment:
Perhaps we need to find a way to put here our official domain instead of private IP address.